public/omni/infrastructure-and-extensions/self-hosted/deploy-omni-on-prem.mdx

```diff
@@ -151,36 +153,9 @@ There are two easy ways to run Omni: docker-compose and a simple `docker run`. W
 
     #### Auth0
 
-    ```bash
-    docker run \
-      --net=host \
-      --cap-add=NET_ADMIN \
-      --device /dev/net/tun \
-      -v $PWD/etcd:/_out/etcd \
-      -v <path to TLS certificate>:/tls.crt \
-      -v <path to TLS key>:/tls.key \
-      -v $PWD/omni.asc:/omni.asc \
-      ghcr.io/siderolabs/omni:<tag> \
-        --account-id=$(uuidgen) \
-        --name=onprem-omni \
-        --cert=/tls.crt \
-        --key=/tls.key \
-        --siderolink-api-cert=/tls.crt \
-        --siderolink-api-key=/tls.key \
-        --private-key-source=file:///omni.asc \
-        --event-sink-port=8091 \
-        --bind-addr=0.0.0.0:443 \
-        --siderolink-api-bind-addr=0.0.0.0:8090 \
-        --k8s-proxy-bind-addr=0.0.0.0:8100 \
-        --advertised-api-url=https://<domain name for onprem omni>/ \
-        --siderolink-api-advertised-url=https://<domain name for onprem omni>:8090/ \
-        --siderolink-wireguard-advertised-addr=<ip address of the host running Omni>:50180 \
-        --advertised-kubernetes-proxy-url=https://<domain name for onprem omni>:8100/ \
-        --auth-auth0-enabled=true \
-        --auth-auth0-domain=<Auth0 domain> \
-        --auth-auth0-client-id=<Auth0 client ID> \
-        --initial-users=<email address>
-    ```
+    <CodeBlock lang="sh">
```

Comment:
This block is really gross. Do we have to write it that way or can we keep line breaks? It'll be easier to maintain.

Reply:
Yes, apparently if I don’t format it this way, Mintlify adds extra indentation that messes up the code block. I asked Edward, and he suggested manually adding each line break and tab so it renders cleaner on the docs page.

Reply:
But ill look for another way